src/services/mongoose/schema.types.d.ts

```diff
@@ -58,12 +58,12 @@ interface UserType {
   earn?: string[]
   deviceToken?: string[]
   excludeCashback?: boolean
-  contacts: ConcatObjectForUser
-  created_at?: string
+  contacts: ConcatObjectForUser[]
```

Comment:
Is this an old typo coming forward? (`ContactObjectForUser`)

Reply:
yup fixed